test/integration/comma-test-csv/comma-test-csv.postman_collection.json

```diff
@@ -16,9 +16,11 @@
             "type": "text/javascript",
             "exec": [
               "var res = JSON.parse(responseBody);",
-              "tests['Stringified number was parsed correctly'] = data.num === 1;",
```

Comment:
this test was removed @codenirvana ?

Reply:
@shamasis It was replaced with an equivalent, as can be seen in the CSV fixture below.